templates/sem_eval_2014_task_1/templates.yaml

```diff
@@ -0,0 +1,52 @@
+dataset: sem_eval_2014_task_1
+templates:
+  14b0f0c7-0026-466f-8d9e-9dc6c32bf111: !Template
+    id: 14b0f0c7-0026-466f-8d9e-9dc6c32bf111
+    jinja: 'Does the premise: "{{premise}}" agree with the hypothesis: "{{hypothesis}}"
+      ? ||| {{["No clear answer", "yes", "no"][entailment_judgment]}}'
+    name: entailment_basic_3
+    reference: ''
+    task_template: false
+  2aa091cb-02ff-4c8c-964c-4c5e53df8c1b: !Template
+    id: 2aa091cb-02ff-4c8c-964c-4c5e53df8c1b
+    jinja: "How related are the two sentences : \"{{hypothesis}}\" and \"{{premise}}\"\
+      \ ? Rate it from 1-5. \n||| {{relatedness_score}}"
+    name: relatedness_basic_2
+    reference: ''
+    task_template: false
+  75203dd2-5ec3-4e91-b95f-228ad9bd2010: !Template
+    id: 75203dd2-5ec3-4e91-b95f-228ad9bd2010
+    jinja: "Sentence 1: \"{{hypothesis}}\" \nSentence 2: \"{{premise}}\"\nAre the\
+      \ two sentences {{\"entailing\"}} or {{\"contradicting\"}} each other?\n|||\
+      \ {{[\"neither\",\"entailing\",\"contradicting\"][entailment_judgment]}}"
+    name: entailment_basic_2
+    reference: ''
+    task_template: false
+  892c58fd-64f5-4059-8fb8-c74bc025ff40: !Template
+    id: 892c58fd-64f5-4059-8fb8-c74bc025ff40
+    jinja: "Given the following hypothesis: {{hypothesis}}.\nAs well as the premise:\
+      \ {{premise}}, \nPredict the Entailment relation between the premise and hypothesis\
+      \ from the labels {{\"Neutral\"}}, {{\"Entailment\"}}, {{ \"Contradiction\"\
+      }} |||\n {{[\"Neutral\", \"Entailment\",  \"Contradiction\"][entailment_judgment]}}\n"
+    name: entailment_basic_1
+    reference: ''
+    task_template: false
+  91a6b1db-be59-41bd-9eea-73bb7a4e7350: !Template
+    id: 91a6b1db-be59-41bd-9eea-73bb7a4e7350
+    jinja: 'Out of {{"neither entails nor contradicts"}}, {{"entails"}}, {{ "contradicts"}},
+      the hypothesis "{{hypothesis}}" ||| {{["neither entails nor contradicts", "entails",  "contradicts"][entailment_judgment]}}
```

Comment:
It's best to only have the model predict one thing - I'm not sure the model will be able to predict the entailment relationship if it's not given the premise.

Reply:
You're right, I'll modify this.